blog/_posts/2017-10-01-what-a-whirlwind.md

```diff
@@ -0,0 +1,67 @@
+
+<br>
+
+## What did we build?
+
+Our project requirements had two components:  
+
+1) A notification center where website admins can create and save notification messages to Worldbrain's 4,000 users.  Notifications can consist of anything from product updates, security alerts and new features.  Notifications are then saved to a mongo db, and made available as an API.  
+
+A react app that renders both a form and list of notifications previously created and sent.  
+
+2) An addition to the chrome extension that shows users there are new notifications via a badge on the extension icon, and list that shows all the notifications, all in React.  All unread notifications are bolded, and when it has been read it then becomes un-bolded.  Each user's read & unread notifications are saved and updated locally in the browser using PouchDB.
+
+![teamImpactdevs](/img/blog/2017/Worldbrain Project.png)
+<div class="image-credits"> What we built this summer</div>
+
+<br>
+
+## How did we build it?
+
+From the beginning, we tried to stay organized and clearly communicate our goals for what we wanted to build and how to build it.  
+
+Before the project began, we created a mockup and a project milestone plan with agreed-upon weekly and monthly goals and objectives with Oliver, our Worldbrain mentor.   
+
+Once the project started, we made a daily log with what we did that day, and anything interested we learned with links.  It's really interesting to refer back to it now!  Each day we had at least one sit-down where we'd ask about each other's progress and maybe even teach each other what we learned or how to build something.  For example, I taught Jess how to build a node server, and she taught me how PouchDb works.
+
+After two months of research, tutorials, building and deploying, we finally built the minimum viable product of our project. For the last month, we refactored, tested and added new features to each component.
+
+<br>
+
+## Lessons and Challenges
+
+One of the best and worst parts of our project is that we were given free reign to build using whatever stack we wanted.  It was very freeing and made us feel like our mentor really trusted us with making the right decision.  However, as a newbie developers sometimes we would spend days or even weeks researching different technologies, not knowing if we could overcome the steep learning curve for using React.  
+
+Anticipating time to build - this was probably the trickiest aspect.  It's really hard to predict and allocate time for building a new page or feature - because a lot can go wrong, and can totally derail the project. For us deploying via Heroku was a huge challenge and took over a week to understand why we it wouldn't work.  But while discussing this with senior developers, even they still face this problem, which made us feel.. slightly better.
```

Comment:
“why we it wouldn't work” => “why it wouldn't work”

Reply:
the last sentence is hard to read, you could add something like “we found out that even they still face this problem” to make it more legible :)